src/grpc/proto/keybroker.proto

```diff
@@ -55,7 +55,18 @@ message SecretResponse {
     string LaunchSecretData = 2;
 }
 
+message OnlineSecretRequest {
```

Comment:
Please add a description of this endpoint in README.md (explaining the expected flow).